### PR TITLE
Icons v1.53.0

### PR DESCRIPTION
**Pull request by jeremyfuksa**
### Summary

**What was changed:**
* Bumped `one-cerner-style-icons` to 1.53.0
  * Added Multiple Results - Corner, Med Tablet - Quarter, and Med Tablet - Quarter Fill

**Why it was changed:**

Consumers are awaiting these icon additions.

### Testing

Attempted WDIO testing, but experienced errors. Still needs screenshot test rebuild.

### Reviews

In addition to engineering reviews, this PR needs:

- [ ] UX review
- [ ] Accessibility review
- [ ] Functional review

---

Thank you for contributing to Terra.
@cerner/terra

### CODE REVIEW

File: packages/terra-core-docs/src/terra-dev-site/doc/icon/example/IconThemeable.jsx [hunk 1]
```diff
@@ -1,250 +1,251 @@
...
-IconXSymbolLight, 
-IconZoomIn, 
-IconZoomOut, 
-} from 'terra-icon';
-
+import IconAdd from 'terra-icon/lib/icon/IconAdd';
+import IconSearch from 'terra-icon/lib/icon/IconSearch';
+import IconCommit from 'terra-icon/lib/icon/IconCommit';
+import IconCheckmark from 'terra-icon/lib/icon/IconCheckmark';
+import IconTrash from 'terra-icon/lib/icon/IconTrash';
```
Comment (sdadn) [inline packages/terra-core-docs/src/terra-dev-site/doc/icon/example/IconThemeable.jsx:8]: Was this intentional or a merge conflict?

Comment (jeremyfuksa) [reply packages/terra-core-docs/src/terra-dev-site/doc/icon/example/IconThemeable.jsx:8]: This would have been auto-built by the migration script.

Comment (sdadn) [reply packages/terra-core-docs/src/terra-dev-site/doc/icon/example/IconThemeable.jsx:8]: Ah okay. That should be an easy fix in a separate PR.

Comment (sdadn) [reply packages/terra-core-docs/src/terra-dev-site/doc/icon/example/IconThemeable.jsx:8]: Opened #4098 to update the scripts.

**Review by sdadn — APPROVED** [review]

File: packages/terra-core-docs/src/terra-dev-site/doc/icon/example/IconThemeable.jsx (hunk 1 above)
Comment (sdadn) [reply packages/terra-core-docs/src/terra-dev-site/doc/icon/example/IconThemeable.jsx]: Merged 4098 & updated this PR.

**Review by adoroshk — APPROVED** [review]

**Pull request merged**